examples/redux/jest.config.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  preset: 'react-native',
```

Comment:
shouldn't we use our preset?
```suggestion
  preset: '@testing-library/react-native',
```